daemon/src/taker_cfd.rs

```diff
@@ -301,6 +312,26 @@ impl Actor {
 
         Ok(())
     }
+
+    // TODO: Duplicated with maker
```

Comment:
if you'd like to reduce duplication, you could extract a function that would look like this when invoked:

`handle_commit(&mut conn, self.wallet, cfd).await?`